lib/Ohmbrewer_Sprouts.cpp

```diff
@@ -9,6 +9,9 @@
 #include "Ohmbrewer_RIMS.h"
 #include "Ohmbrewer_Onewire.h"
 
+
+//TODO onewire index refactor ??
+
```

Comment:
TODO:
1) Refactor ```Ohmbrewer::Sprouts::parseTemperatureSensorPins``` so that instead of ```busPin``` it has ```probeIndex``` (just a rename, really)
2) Refactor ```Ohmbrewer::Sprouts::addTemperatureSensor``` so that instead of ```busPin``` it has ```probeIndex``` (just a rename, really). Then, change the TemperatureSensor constructor it calls is ```TemperatureSensor(int ID)```.
3) Refactor the ```TemperatureSensor(int ID)``` constructor to automatically use the ID as the ```probeIndex``` in the ```Onewire(int probeIndex)``` constructor for its probe.
4) Refactor ```Ohmbrewer::Sprouts::parseThermostatPins``` so that instead of ```sensorBus``` it has ```probeIndex``` (just a rename, really)
5) Refactor ```Ohmbrewer::Sprouts::parseRIMSPins``` so that it expects the Safety Sensor's probeIndex in between the Thermostat pins and the Pump pins. You can pretty much lift the sensor code block from ```Ohmbrewer::Sprouts::parseThermostatPins``` and rename stuff and you'll probably be in business there. Once you've added that block, you'll need to add another parameter to the function to return the parsed value. The format of it will be similar to how the pump pin gets returned.
6) Refactor ```Ohmbrewer::Sprouts::addRIMS``` to handle the safety sensor ```probeIndex``` being returned from ```Ohmbrewer::Sprouts::parseRIMSPins```. You'll need to take the returned value and plug it into the right place in the RIMS constructor, naturally.
7) Refactor the RIMS and Thermostat constructors so that they can handle the ```probeIndex```'es appropriately. It looks like you've already started on this (or at least noted it).
8) Update the README/docs to include the new argument in the argument strings for /add.

I think if you do all of those things, we'll be looking pretty good. That should get /add working with the IDs/Indices and the other Particle.functions call everything by Type+ID pairs so we shouldn't need to do anything to fix them.